app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,136 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+html {
+  background: rgb(0, 22, 47);
+  background: linear-gradient(0deg, rgba(0, 22, 47, 1) 0%, rgba(55, 120, 194, 1) 100%);
+  width: 100.2vw;
+  height: 100vh;
+  overflow-x: hidden;
+}
+
+#landing {
+  width: 100%;
+  height: 100%;
+  margin-top: 30%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#landing h1 {
+  color: white;
+  font-family: 'Pacifico', cursive;
+  font-size: 4em;
+}
+
+#landing a {
+  color: white;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
```

Comment:
Great designs so far. 👏 

Kindly note the required font families for the headers and body of your design.

![image](https://user-images.githubusercontent.com/27709832/221377239-cc94ce44-ef55-4821-bb9c-4b83fac8b15f.png)